packages/common/src/lib/badge-icon/badge-icon.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { MatBadgeIconDirective } from './badge-icon.directive';
+import { MatBadgeModule, MatIconModule } from '@angular/material';
+
+@NgModule({
+  imports: [MatBadgeModule, MatIconModule],
+  declarations: [MatBadgeIconDirective],
+  exports: [MatBadgeIconDirective]
+})
+export class IgoMatBadgeIconModule {
+  static forRoot(): ModuleWithProviders {
```

Comment:
I wouldn't make it a ModuleWithProviders if it's not required.

Reply:
fixed